test/track.js

```diff
@@ -72,5 +75,43 @@ exports.track = {
             test.equal(e, undefined, "error should be undefined");
             test.done();
         });
+    },
+
+    "calls `track` endpoint if within last 5 days": function(test) {
+        var expected_endpoint = "/track",
+            expected_data = {
+                event: 'test',
+                properties: {
+                    token: 'token',
+                    time: mockNowTime / 1000
+                }
+            };
+
+        this.mixpanel.send_request.callsArgWith(2, undefined);
+
+        test.expect(1);
+        this.mixpanel.track("test", function(e) {
+            test.equal(e, undefined, "error should be undefined");
+            test.done();
+        });
```

Comment:
For your two new tests, don't you want to be checking
```js
this.mixpanel.send_request.calledWithMatch(expected_endpoint, expected_data)
```
? See the "calls send_request with correct endpoint and data" test case above.